src/scripts.js

```diff
@@ -178,32 +190,41 @@ function getDataToPost(event) {
   const calenderDate = calenderInput.value.split("-").join("/");
   inputValue.value = calenderInput.value;
   dateInput = calenderDate;
+  let data;
+  let detail;
   if (result === "Sleep Data") {
-    let data = {
+    data = {
       userID: user.id,
       date: calenderDate,
       hoursSlept: parseInt(hoursSlept.value),
       sleepQuality: parseInt(sleepQuality.value),
     };
-    postData("sleep", data);
+    detail = "sleep";
   } else if (result === "Hydration Data") {
-    let data = {
+    data = {
       userID: user.id,
       date: calenderDate,
       numOunces: parseInt(numOfOunces.value),
     };
-    postData("hydration", data);
+    detail = "hydration";
   } else if (result === "Activity Data") {
-    let data = {
+    data = {
       userID: user.id,
       date: calenderDate,
       numSteps: parseInt(minsActive.value),
       minutesActive: parseInt(numOfSteps.value),
       flightsOfStairs: parseInt(flightsOfStairs.value),
     };
-    postData("activity", data);
+    detail = "activity";
   }
-  promiseAll();
+  console.log(detail);
```

Comment:
looks like we got some console logs here

Reply:
done!